src/index.js

```diff
@@ -1,23 +1,24 @@
 import { parquetRead } from './read.js'
 
 export { parquetMetadata, parquetMetadataAsync, parquetSchema } from './metadata.js'
-export { parquetRead }
 export { parquetQuery } from './query.js'
 export { snappyUncompress } from './snappy.js'
 export { asyncBufferFromUrl, byteLengthFromUrl, cachedAsyncBuffer, flatten, toJson } from './utils.js'
+export { parquetRead }
 
 /**
  * This is a helper function to read parquet row data as a promise.
  * It is a wrapper around the more configurable parquetRead function.
- *
- * @param {Omit<ParquetReadOptions, 'onComplete'>} options
- * @returns {Promise<Record<string, any>[]>} resolves when all requested rows and columns are parsed
+ * @import {BaseParquetReadOptions} from '../src/types.d.ts'
+ * @param {BaseParquetReadOptions} options
+ * @returns {Promise<Record<string, any>[]>} resolves when all requested rows and columns are parsed.
+ *   Depending on the rowFormat, this will be an array of arrays or an array of objects. The default is an array of objects.
 */
 export function parquetReadObjects(options) {
   return new Promise((onComplete, reject) => {
     parquetRead({
-      rowFormat: 'object',
       ...options,
+      rowFormat: 'object',
```

Comment:
This is a breaking change. Before, the function could be called with `rowFormat: 'array'`, and the return type would be `any[][]` (meaning that the function type was wrong).

I guess people would generally not pass `rowFormat: 'array'` as the name of the function explicitly says "objects", the return type says objects, and the [README](https://github.com/hyparam/hyparquet?tab=readme-ov-file#parquetreadobjects) showcases objects.

But, theoretically, it's still a breaking change.